docs/index.md

```diff
@@ -72,6 +72,17 @@ embeddings for many languages. Here's a list of resources;
 
 - [AraVec](https://github.com/bakrianoo/aravec#download) has embeddings for Arabic trained on twitter and/or Wikipedia.
 
+## **Sparse Featurizers**
+
+### SemanticMapFeaturizer
+
+**`rasa_nlu_examples.featurizers.sparse.SemanticMapFeaturizer` [docs](docs/featurizer/semantic_map/)**
+
+The `SemanticMapFeaturizer` is an experimental *sparse* featurizer developed by Rasa.
+It can only be used in combination with pre-trained embedding files, which you can
+find [here](https://github.com/RasaHQ/rasa-embeddings/tree/main/embeddings/semantic_map).
+Please refer to our [blog posts](blog.rasa.com/exploring-semantic-map-embeddings-1/) for more details.
```

Comment:
1. Is this blog live yet? 
2. Let's add https. blog.rasa.com/exploring-semantic-map-embeddings-1/ -> https://blog.rasa.com/exploring-semantic-map-embeddings-1/